node/internal_binding/handle_wrap.ts

```diff
@@ -38,11 +38,11 @@ export class HandleWrap extends AsyncWrap {
   }
 
   ref() {
-    notImplemented("HandleWrap.prototype.ref");
+    unreachable();
```

Comment:
Why are these unreachable?

Reply:
As far as can see currently these methods exist purely for defining an interface on classes rather than something that will ever be implemented - felt more accurate to fail with unreachable than not implemented.

Wonder whether they should be defined at all… or if we want them, make use of the inheritance better and not override in all of the other wraps